hack/update-doc-stack-version.sh

```diff
@@ -1,29 +0,0 @@
-#!/usr/bin/env bash
-
-# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
-# or more contributor license agreements. Licensed under the Elastic License;
-# you may not use this file except in compliance with the Elastic License.
-
-#
-# Update the Elastic stack version in samples and documentation.
-# Usage: ./hack/update-doc-stack-version.sh <version>
```

Comment:
I think we should not delete this script because this PR only deals with the version update in the documentation and not in the [samples](https://github.com/elastic/cloud-on-k8s/tree/master/config/samples).

Reply:
Thanks for pointing it out. I overlooked the part about updating samples.